package.json

```diff
@@ -118,11 +116,18 @@
   },
   "browserslist": [
     "last 1 Chrome version",
-    "last 1 Android version",
     "last 1 Firefox version",
-    "last 1 FirefoxAndroid version",
-    "last 1 Safari version",
-    "last 1 iOS version",
     "last 1 Edge version"
-  ]
+  ],
+  "release-it": {
+    "github": {
+      "release": false
+    },
+    "npm": {
+      "publish": false
+    },
+    "git": {
+      "requireCleanWorkingDir": true
```

Comment:
I think this is also the default behavior? https://github.com/release-it/release-it/blob/master/docs/git.md#clean-working-directory